services/app-api/forms/2026/elements.ts

```diff
@@ -296,3 +297,49 @@ export const areYouReportingStratification: RadioTemplate = {
     { label: "No", value: "no" },
   ],
 };
+
+export const shortTermNestedHeading: NestedHeadingTemplate = {
+  type: ElementType.NestedHeading,
+  id: "short-term-subheader",
+  text: "Short Term",
+};
+
+export const mediumTermNestedHeading: NestedHeadingTemplate = {
+  type: ElementType.NestedHeading,
+  id: "medium-term-subheader",
+  text: "Medium Term",
+};
+
+export const longTermNestedHeading: NestedHeadingTemplate = {
+  type: ElementType.NestedHeading,
+  id: "long-term-subheader",
+  text: "Long Term",
+};
+
+export const denominatorAutoCalc18to64: TextboxTemplate = {
+  type: ElementType.Textbox,
+  id: "denom-auto-calc-18-to-64",
+  helperText: "Auto-calculates",
+  label: "Denominator (18 to 64 years)",
```

Comment:
Could we pass in age range as a prop instead and refactor the other elements across the measure page that are basically the same but the only different language is `age range` vs `term stay`? Although I'm pretty sure @ailZhou is already covering this in Rate Calculations PR... & @benmartin-coforma too